build.gradle

```diff
@@ -20,22 +20,19 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile) {
 
 dependencies {
     // BOM with external dependencies restrictions
-    api platform('com.exactpro.th2:bom:3.0.0')
+    api platform('com.exactpro.th2:bom:4.0.2')
 
     // the library for interaction with infra API
-    implementation 'com.exactpro.th2:common:3.13.6'
+    implementation 'com.exactpro.th2:common:3.42.0'
 
     // Logging. All versions are specified in BOM dependency
-    implementation "org.slf4j:slf4j-log4j12"
     implementation "org.slf4j:slf4j-api"
 
     // Idiomatic logging for Kotlin. Wraps slf4j
-    implementation 'io.github.microutils:kotlin-logging:2.0.4'
+    implementation 'io.github.microutils:kotlin-logging:3.0.4'
```

Comment:
This version uses kotlin 1.7. The last version that uses kotlin 1.6 is `3.0.0`
https://mvnrepository.com/artifact/io.github.microutils/kotlin-logging/3.0.0